package.json

```diff
@@ -24,7 +24,8 @@
     "error-code-doc:check": "npm run error-code-doc && git diff --exit-code",
     "hints-doc": "ts-node ./composition-js/src/genHintDoc.ts > ./docs/source/hints.md",
     "hints-doc:check": "npm run hints-doc && git diff --exit-code",
-    "gen-test-supergraph": "ts-node ./query-planner-js/src/__tests__/genTestSupergraph.ts"
+    "gen-test-supergraph": "ts-node ./query-planner-js/src/__tests__/genTestSupergraph.ts",
+    "changeset-version": "changeset version && npm i"
```

Comment:
It's ok there isn't a `npx` here? You're not relying on changesets being globally installed?

Reply:
Yeah. I'm either not searching for the right thing or documentation isn't clear but scripts can run locally installed binaries like this when they're installed as dependencies in the project.